src/core/structured-log.js

```diff
@@ -277,12 +277,15 @@
       invoke('TRACE', mt, arguments);
     };
 
-    self.information = function(messageTemplate) {
+    self.verbose = self.trace;
```

Comment:
There was a comment earlier that debug should be a different level to verbose/trace. My preference would be to drop the 'trace' and just separate levels for debug and verbose.